tests/install.bats

```diff
@@ -115,6 +115,24 @@ SUITE_NAME=$( test_suite_name )
   assert_lib_installed Alpha-1.0 "${tag_alpha_lib_revision_hash}"
 }
 
+@test "${SUITE_NAME}: existing library resets upstream" {
+  create_decomposer_json alpha_tag_version
+
+  create_repository alpha-lib
+
+  # create usual clone of library
+  git clone "${TEST_REPOS_DIR}/alpha-lib" "${DECOMPOSER_TARGET_DIR}/Alpha-1.0"
+
+  # create new commit in repository and tag
+  git -C "${TEST_REPOS_DIR}/alpha-lib" remote set-url origin https://moveagency.com
+
+  run_decomposer install
+  [ "${status}" -eq 0 ]
+  [ "${lines[0]}" = "Installing Alpha...done" ]
+
+  assert_lib_installed Alpha-1.0 "${tag_alpha_lib_revision_hash}"
```

Comment:
This just verifies the hash is as expected, but we only have the commits from the original clone. To properly test this `decomposer install` would have to fetch new commits, **and** we'd have to verify that the remote after running `decomposer install` matches what's in `decomposer.json`